checkov/terraform/tf_parser.py

```diff
@@ -420,6 +505,14 @@ def add_tfvars(self, module: Module, source: str) -> None:
                 block = [{var_name: {"default": default}}]
                 module.add_blocks(BlockType.TF_VARIABLE, block, path, source)
 
+    def add_tfvars_with_source_dir(self, module: Module, source: str, source_dir: str) -> None:
+        if not self.external_variables_data:
+            return
+        for (var_name, default, path) in self.external_variables_data:
```

Comment:
```suggestion
        for var_name, default, path in self.external_variables_data:
```
I don't think you need the brackets